wsd/LOOLWSD.cpp

```diff
@@ -3995,7 +3995,7 @@ int LOOLWSD::innerMain()
     }
 
     if (Log::logger().getLevel() >= Poco::Message::Priority::PRIO_INFORMATION)
-        LOG_ERR("Log level is set very high to '" << LogLevel << "' this will have a "
+        LOG_WRN("Log level is set very high to '" << LogLevel << "' this will have a "
```

Comment:
I think this is done intentionally as ERR so that it is caught by admins (assuming they do read the logs during a normal operation (i.e. when there are no complaints).

I suggest we make this conditional. #ifndef _DEBUG so that it doesn't cause noise for developers (for example developers grepping for ERR would not find this helpful), but otherwise leave it as ERR.

Reply:
Right - I added this myself. We saw systems in production that had logging set to stupidly high levels which totally cripples performance; as the git log shows =)